service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -63,4 +71,14 @@ private void run3037(CaseDetails caseDetails) {
         data.put("additionalApplicationsBundle", additionalApplicationsBundles);
         caseDetails.setData(data);
     }
+
+    private void run3087(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
```

Comment:
it will set issue date as date when migration was run, but maybe that is ok

Reply:
checked with jason all good to replace date of issue